daffodil-runtime1/src/main/scala/org/apache/daffodil/debugger/InteractiveDebugger.scala

```diff
@@ -1842,6 +1908,32 @@ class InteractiveDebugger(runner: InteractiveDebuggerRunner, eCompilers: Express
         }
       }
 
+      object SetDiffExcludes extends DebugCommand {
+        val name = "diffExcludes"
+        val desc = "set info commands to exlude in the 'info diff' commanad"
```

Comment:
Fix typo in "exlude".

Reply:
Will fix.

Codecov seems a bit flakey to me. Sometimes comments/inline code shows up, sometimes they don't. And looking at the pull request there are a few lines that aren't covered:

https://app.codecov.io/gh/apache/daffodil/compare/493/diff

I'll see if I can tweak this test or add a new one to increase the coverage.

Reply:
Hmm, I couldn't view any report data in your 493 link (the page said Unauthorized Github API: Forbidden Check on Codecov's status or see our docs for common support: Error 403).  I tried a report link from one of the build jobs' logs but the diff tab said "No files changed" so I couldn't see per-line coverage there either.  Not sure what's going on.  Here's the report link I tried.

https://codecov.io/gh/apache/daffodil/commit/ba2d93f012bdff346c6f380a180c7ea3253bb5fb

Reply:
Yes, you're right, I had to log into codecov on their home page.  Now both links work and show me line by line coverage changes although one aggregates all build jobs and the other is from only one build job.